Packages/com.unity.inputsystem/InputSystem/Plugins/EnhancedTouch/Touch.cs

```diff
@@ -42,212 +60,245 @@ public struct Touch : IEquatable<Touch>
+        /// a device control.</remarks>
         public Vector2 startScreenPosition => state.startPosition;
 
         /// <summary>
         /// Screen-space motion delta of the touch.
         /// </summary>
-        /// <value>Screen-space motion delta of the touch.</value>
         /// <remarks>
+        /// <para>
         /// Note that deltas have behaviors attached to them different from most other
         /// controls. See <see cref="Pointer.delta"/> for details.
+        /// </para>
+        /// <para>
+        /// Also see <see cref="TouchControl.delta"/> for retrieving delta directly from a device control.
+        /// </para>
         /// </remarks>
-        /// <seealso cref="TouchControl.delta"/>
         public Vector2 delta => state.delta;
 
         /// <summary>
         /// Number of times that the touch has been tapped in succession.
         /// </summary>
         /// <value>Indicates how many taps have been performed one after the other.</value>
         /// <remarks>
+        /// <para>
         /// Successive taps have to come within <see cref="InputSettings.multiTapDelayTime"/> for them
         /// to increase the tap count. I.e. if a new tap finishes within that time after <see cref="startTime"/>
         /// of the previous touch, the tap count is increased by one. If more than <see cref="InputSettings.multiTapDelayTime"/>
         /// passes after a tap with no successive tap, the tap count is reset to zero.
+        /// </para>
+        /// <para>
+        /// Also see <see cref="TouchControl.tapCount"/> for retrieving tap count directly from a device control.
+        /// </para>
         /// </remarks>
-        /// <seealso cref="TouchControl.tapCount"/>
         public int tapCount => state.tapCount;
 
         /// <summary>
         /// Whether the touch has performed a tap.
         /// </summary>
         /// <value>Indicates whether the touch has tapped the screen.</value>
         /// <remarks>
+        /// <para>
         /// A tap is defined as a touch that begins and ends within <see cref="InputSettings.defaultTapTime"/> and
         /// stays within <see cref="InputSettings.tapRadius"/> of its <see cref="startScreenPosition"/>. If this
         /// is the case for a touch, this button is set to 1 at the time the touch goes to <see cref="phase"/>
         /// <see cref="TouchPhase.Ended"/>.
-        ///
+        /// </para>
+        /// <para>
         /// Resets to 0 only when another touch is started on the control or when the control is reset.
+        /// </para>
+        /// <para>
+        /// Use <see cref="tapCount"/> to determine if there were multiple taps occurring during the frame.
+        /// Also note that <see cref="TouchControl.tap"/> may be used to determine whether there was a tap.
+        /// </para>
         /// </remarks>
-        /// <seealso cref="tapCount"/>
-        /// <seealso cref="InputSettings.defaultTapTime"/>
```

Comment:
might be worth including defaultTapTime in the remarks too

Reply:
It already is? See first line in remarks.